core/src/main/java/curso/modelo/Curso.java

```diff
@@ -1,30 +1,35 @@
 package curso.modelo;
 
+import curso.exception.ExceptionAtributosCursoIncorrectos;
+
 import java.time.LocalDate;
 import java.util.UUID;
 
 public class Curso {
     private UUID id;
     private String nombre;
     private LocalDate fecha_cierre_inscripcion;
```

Comment:
Por convención, sugiero renombrar estas variables Java para evitar el uso de guión bajo. Cambiar a` lowerCamelCase`
```suggestion
    private LocalDate fechaCierreInscripcion;
```
ℹ️ Aplica a todas las variables que revistan la misma situación